.github/dependabot.yml

```diff
@@ -0,0 +1,56 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/dvp"
+  schedule:
+    interval: monthly
+  open-pull-requests-limit: 10
+  target-branch: develop
+  ignore:
```

Comment:
Can you please check if we still need the ignore clause as we are using latest version of `packaging`.